Sources/NFCPassportReader/TagReader.swift

```diff
@@ -16,24 +16,29 @@ import CoreNFC
 public class TagReader {
     var tag : NFCISO7816Tag
     var secureMessaging : SecureMessaging?
-    var maxDataLengthToRead : Int = 0xA0  // Should be able to use 256 to read arbitrary amounts of data at full speed BUT this isn't supported across all passports so for reliability just use the smaller amount.
-
+    var maxDataLengthToRead : Int
+    var useExtendedMode : Bool
     
     weak var trackingDelegate: PassportReaderTrackingDelegate?
     
     var bytesRead : ((Int)->())?
 
-    init( tag: NFCISO7816Tag, trackingDelegate: PassportReaderTrackingDelegate? ) {
+    init( tag: NFCISO7816Tag, trackingDelegate: PassportReaderTrackingDelegate?, useExtendedMode: Bool ) {
```

Comment:
This is a bit unusual way to format the code with spaces around parameters 😄 But I see that this formatting style is used in other places as well, therefore I'm fine with leaving it as is 👍

Reply:
100% agree with you, the formatting of the project is "interesting" - to put it nicely. 😅